app/models/batch.rb

```diff
@@ -51,9 +51,10 @@ class Item
     #   @return [String]
     # @!attribute id [rw]
     #   @return [String]
-    # @!attribute object [rw]
-    #   @return [ActiveFedora::Base, String]
+    # @!attribute object [w
+    #   @return [ActiveFedora::Base, nil]
     attr_accessor :batch_id, :id, :object, :store
+    attr_writer   :object
```

Comment:
I'm don't quite get why :object appears in both 56 and 57 - doesn't :object in line 56 imply there's both reader and writer (for all of the attributes in 56)?

Reply:
it should have been removed from accessors, thanks!